serverless-haskell.hsfiles

```diff
@@ -190,7 +190,7 @@ plugins:
 #
 # resolver: ./custom-snapshot.yaml
 # resolver: https://example.com/snapshots/2018-01-01.yaml
-resolver: lts-16.3
+resolver: lts-17.4
```

Comment:
Why is this 17.4 when `stack.yaml` uses 17.7?

Reply:
Fixed!